users/danielo515/process_records.c

```diff
@@ -44,124 +44,157 @@ void refresh_incremental_macros(uint16_t kc) {
 }
 // Send control or GUI depending if we are on windows or mac
 bool CMD(uint16_t kc) {
-    if(onMac){ tap_code16(LGUI(kc)); } else { tap_code16(LCTL(kc)); }
+    if (onMac) {
```

Comment:
in theory, you could use mod_config and keycode_config to switch stuff based on bootmagic/magic settings.

Reply:
Yeah, we already had this conversation 😄 do you remember it? It does not fit all my use-cases. I'll take a second look anyway if it has changed since then.

Reply:
Now that you mention it, yes, I do. Sorry! 
(I has memory of goldfish, on a good day)